demo.md

```diff
@@ -12,4 +12,4 @@ toc: false
 
 <script type="text/javascript" src="https://asciinema.org/a/48474.js" id="asciicast-48474" async></script>
 
-<a href="https://docs.pulpproject.org/user-guide/installation/index.html" class="btn btn-primary">Install Pulp</a>
\ No newline at end of file
+<a href="https://docs.pulpproject.org/en/2.21/user-guide/installation/index.html" class="btn btn-primary">Install Pulp</a>
```

Comment:
Do we want to have Pulp 2 or Pulp 3 install button here?  If 3, we should probably update the associated video as well?

Reply:
I'm hoping to redo this page in a couple weeks with demo videos and will also update the link here. Until then, I think this is fine.

Reply:
Or better yet, this could say something like "We're working on demo videos for Pulp 3.0. Please check back for updates."